src/meta/processors/admin/HBProcessor.cpp

```diff
@@ -43,8 +43,8 @@ void HBProcessor::process(const cpp2::HBReq& req) {
        }
     }
 
-    VLOG(3) << "Receive heartbeat from " << host
-            << ", role = " << apache::thrift::util::enumNameSafe(req.get_role());
+    LOG(INFO) << "Receive heartbeat from " << host
```

Comment:
VLOG（1）？

Reply:
Previously was INFO, modified to VLOG(3) by others. It is the most convenient way to tell which is meta leader.